app/web/template/layout/page.gohtml

```diff
@@ -16,8 +16,8 @@
       <link rel="apple-touch-icon" sizes="167x167" href="/static/assets/images/govuk-apple-touch-icon-167x167.png" >
       <link rel="apple-touch-icon" sizes="152x152" href="/static/assets/images/govuk-apple-touch-icon-152x152.png" >
       <link rel="apple-touch-icon" href="/static/assets/images/govuk-apple-touch-icon.png" >
-      <link href="/static/stylesheets/all.css" rel="stylesheet">
-      <script async src="/static/javascript/main.js"></script>
+      <link href="/static/stylesheets/all.css?{{ .App.StaticHash }}" rel="stylesheet">
+      <script async src="/static/javascript/main.js?{{ .App.StaticHash }}"></script>
```

Comment:
I was about to comment is this required until I Googled preload and got the answer to my Q 😂 - will have to keep this in mind for future work.